src/v0/util/index.js

```diff
@@ -1575,6 +1575,38 @@ const simpleProcessRouterDest = async (inputs, singleTfFunc, reqMetadata) => {
   );
   return respList;
 };
+/**
+ * This is the sync version of simpleProcessRouterDest
+ * 
+ * @param {*} inputs 
+ * @param {*} singleTfFunc 
+ * @param {*} reqMetadata 
+ * @param {*} processParams 
+ * @returns 
+ */
+const simpleProcessRouterDestSync = async (inputs, singleTfFunc, reqMetadata, processParams) => {
```

Comment:
Consider this that these are the following series of identify events
```[
  { userId: A, trait1: value1, trait2: value2},
   { userId: B, trait1: value1, trait2: value2},
    { userId: B, trait1: value1.1, trait2: value2.1},
      { userId: B, trait1: value1.1, trait2: value2},
    ]
```
So the final state of B user in Braze is `trait1: value: 1.2, trait2: vlaue:2` So when we deduplicate the flow for B should be
1. 
lookup for B --> No user found
 process it and store the data in `store` in case there is no data found from braze -- transformed payload  `{ userId: B, trait1: value1.1, trait2: value2.1}`
 
again lookup B --> No user found
 process it and now deduplicate from `store` and update it -- transformed payload  `{ userId: B, trait1: value1.1, trait2: value2.1}`
 
again same flow
 process deduplicate transformed payload -- `{ userId: B trait2: value2}`
 
 
 So the total output from transformer becomes
 ```[
  { userId: A, trait1: value1, trait2: value2},
   { userId: B, trait1: value1, trait2: value2},
    { userId: B, trait1: value1.1, trait2: value2.1},
      { userId: B, trait2: value2},
    ]
```

if we do async this order might not be guaranteed